source/documentation/tools.md

```diff
@@ -30,14 +30,14 @@ There are a number of tools available for both R and Python to support package m
 
 For R:
 
-* **conda** - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
-* **packrat** - not recommended for R, but remains an option.
+* [**conda**](#conda) - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
+* [**packrat**](#packrat) - not recommended for R, but remains an option.
 
 For Python:
 
-* **venv** and **pip** - Recommended for Python in Analytical Platform, because it's easier and more reliable than conda.
-* **conda environment** installing packages with just **conda** - not recommended, but it might help when you have a package package with a C extension that pip struggles with compiling or installing the binaries
-* **conda environment** installing packages with **conda** and **pip** - not recommended, but gives you the broadest range of package install options. However conda and pip don't play well together - use at your own risk!
+* [**venv** and **pip**](#venv-and-pip) - Recommended for Python in Analytical Platform, because it's easier and more reliable than conda.
```

Comment:
Also, `conda` is not at all widespread. Everyone in the wider Python ecosystem uses `pip` and `venv` so there's also more chance that problems will be fixed or gotchas explained on SO and elsewhere.

Reply:
This is another good point not to use it.

However overall I think we'd be wise not to write off conda. I can't write it off as an option, as it may yet solve install issues, for our users who have neither root access or usually the skills to work around package install problems. 

I think [this hackernews pair of comments](https://news.ycombinator.com/item?id=23380113) speaks for many in the data science world:

> In Python at least pip is a standard that always works to install, even if it doesn't solve the other problems. 

> Unless you're in data  science. Then you'll be split between conda and pip

That discussion also has bits about why wheels aren't perfect.

Reply:
I've added the point about community size 👍